rclcpp/plugins/polygon_base/CHANGELOG.rst

```diff
@@ -0,0 +1,3 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Is this supposed to be during release process? we are planning to release this package, correct?